dvc/dependency/__init__.py

```diff
@@ -49,9 +49,15 @@
 del SCHEMA[BaseOutput.PARAM_METRIC]
 SCHEMA.update(RepoDependency.REPO_SCHEMA)
 SCHEMA.update(ParamsDependency.PARAM_SCHEMA)
+SCHEMA.update({BaseOutput.PARAM_FILTER: str})
 
 
 def _get(stage, p, info):
+    if isinstance(p, dict):
+        p = list(p.items())
+        assert len(p) == 1
```

Comment:
maybe assert not required (should be handled by schema)?

Reply:
Other CLI commands can create/load dependencies, skipping the schema. Good to have an assert.